EIPS/eip-5516.md

```diff
@@ -0,0 +1,190 @@
+
+## Rationale
+
+### SBT as an extension to EIP-1155
+We believe that Soulbound Token serves as a specialized subset of the existing [EIP-1155](./eip-1155.md) tokens. The advantage of such design is the seamless compatibility of Soulbound Token with existing NFT services. Service providers can treat SBTs like NFTs and do not need to make drastic changes to their existing codebase.
+
+Making the standard compatible with [EIP-1155](./eip-1155.md) also allows for SBT binding to Smart Contracts,allowing for a whole new world of possibilities.
+
+### Double-Signature
+The Double-Signature functionality was implemented to prevent the receipt of unwanted tokens. It symbolizes a handshake between the token receiver and token sender, implying that **both** parties agree on the token transfer.
+
+### Metadata.
+The [EIP-1155](./eip-1155.md#metadata) Metadata Interface was implemented for more compatibility with [EIP-1155](./eip-1155.md).
+
+### Guaranteed log trace
+> As the Ethereum ecosystem continues to grow, many DApps are relying on traditional databases and explorer API services to retrieve and categorize data. The EIP-1155 standard guarantees that event logs emitted by the smart contract will provide enough data to create an accurate record of all current token balances. A database or explorer may listen to events and be able to provide indexed and categorized searches of every EIP-1155 token in the contract.
+
+_quoted from [EIP-1155](./eip-1155.md#guaranteed-log-trace)_
+
+This EIP extends this idea to the Double Signature functionality: The `{TokenClaimed}` event logs all the necessary information of a `ClaimOrReject(...)` or `ClaimOrRejectBatch(...)` function call, storing relevant information about the actions performed by the user. This also applies to the `batchTransfer(...)` function: It emits the `{TransferMulti}` event and logs necessary data.
+
+### Exception handling
+Given the non-transferability property of SBTs, if a user keys to an account or a contract get compromised or rotated, a user may lose the ability to associate themselves with the token.
+
+**Given the multi-owner characteristic of [EIP-1155](./eip-1155.md) compliant interfaces and contracts, this allows for SBTs to bind to multiple accounts, providing a potential solution to the issue.**
+
+Multi-owned SBTs can also be issued to a contract account that implements a multi-signature functionality (As recommended in [EIP-4973](./eip-4973.md#exception-handling)), this can be achieved via the [EIP-1155](./eip-1155.md#erc-1155-token-receiver) Token Receiver interface.
+
+### Multi-token
+The multi-token functionality was used to ease organization. All emitted tokens are stored in the same contract, preventing redundant bytecode from being deployed to the blockchain. It also facilitates transfer to token issuers, since all issued tokens are stored and can be accessed under the same address. It is also easy to implement non-fungible and semi-fungible tokens in the same contract.
```

Comment:
```suggestion
The multi-token functionality was used to ease organization. All emitted tokens are stored in the same contract, preventing redundant bytecode from being deployed to the blockchain. It also facilitates transfer to token issuers, since all issued tokens are stored and can be accessed under the same address. It is also easy to implement non-fungible and semi-fungible tokens in the same contract.
```
Defile "ease organization.